src/main/java/util/TickingTimerManager.java

```diff
@@ -0,0 +1,65 @@
+package util;
+
+import ui.UI;
+
+/**
+ * This class is used to hold the TickingTimer instance and perform operations associated with it.
+ */
+public class TickingTimerManager {
```

Comment:
Class's header comment also preferably describe what it represents instead of how it's used.